src/mixs/schema/ancient.yml

```
@@ -438,6 +455,143 @@ slots:
     slot_uri: MIXS:999999922
     range: string
     string_serialization: "{text}"
+  palaeopath_status:
+    annotations:
+      Expected_value: description of health related observation on ancient remains
+    description: >-
+      Describe briefly any relevant palaeopathological or health-related observations of the remains of the individual under study.
+    title: palaeopathology status.
+    examples:
+      - value: "Osteoporosis."
+      - value: "Parasites found in pelvic area."
+      - value: "Caries on right upper molar."
+    in_subset:
+      - nucleic acid sequence source
+    keywords:
+      - palaeopathology
+      - host health
+      - ancient
+    slot_uri: MIXS:999999905 #Not assigned
+    multivalued: false
+    range: string
+    string_serialization: "{text}"
+    required: false
+    recommended: true
```

```suggestion
    recommended: false
```

I wouldn't recommend this, you can't put apply this to seidment for example, setting both to false will basically make it 'optional'